tests/specs/statements/sheBang/SheBang_All.txt

```diff
@@ -53,7 +53,6 @@
 
 [expect]
 #!/foo/bar
-
```

Comment:
Hmmm

Reply:
Damn. This is an swc bug based on my fix.

Reply:
Yikes, sorry about that 😬